### ISSUE TITLE
Finds exact in ios does not return an array

### ISSUE THREAD

**Issue by slipy12** [appium/ruby_lib#423]
In code: 
```
    # Find all elements exactly matching value
    # @param value [String] the value to search for
    # @return [Array<Element>]
    def finds_exact(value)
      if automation_name_is_xcuitest?
        find_ele_by_attr '*', '*', value
      else
        eles_by_json_visible_exact '*', value
      end
    end
```

Should be:

  ```
    # Find all elements exactly matching value
    # @param value [String] the value to search for
    # @return [Array<Element>]
    def finds_exact(value)
      if automation_name_is_xcuitest?
        find_eles_by_attr '*', '*', value       <-- Changed
      else
        eles_by_json_visible_exact '*', value
      end
    end
```
Doing PR

**Comment by KazuCocoa** [comment]
I published 9.1.1.
https://github.com/appium/ruby_lib/releases/tag/v9.1.1

**Comment by KazuCocoa** [comment]
@slipy12
